### DISCUSSION TITLE
Color scheme for the tool panel?

### DISCUSSION THREAD

**Discussion by dannon** [galaxyproject/galaxy#17846] [Ideas]
From the gcc-2021 cofest, thanks @jrr-cpt!

The tool panel uses very light gray colors that are hard to tell apart. Perhaps more visually distinct shading or colors can be used? This may be possible already, or part of the themes that were introduced during this conference.

**Comment by hexylena** [comment]
> Do you have thoughts for how you'd want the colour scheme to work? A random(ish) colour for every tool group? Specific colours for specific ones? Is it a manual process for choosing the colours, or, do you do it dynamically/automatically?

@jrr-cpt responded:

If the default colors just had more contrast, that would be a great start.
Perhaps a set of 3-4 color options with preset colors selected for high contrast would give admins an option for their choice.
